src/pages/CommitDetailPage/subRoute/IndirectChangesTab/IndirectChangesTable/IndirectChangesTable.spec.jsx

```diff
@@ -341,4 +305,40 @@ describe('IndirectChangesTable', () => {
       expect(commitFileDiff).toBeInTheDocument()
     })
   })
+
+  describe('flags and components param in url is set', () => {
+    it('renders flags and/or components no files error message', async () => {
+      const { queryClient } = setup({
+        __typename: 'ImpactedFiles',
+        results: [],
+      })
+      const path = `/gh/codecov/cool-repo/commit/123/indirect-changes${qs.stringify(
```

Comment:
can you add components to the path params so we can test if it's actually being set/so it matches the description of the block